File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res = []

        cnt = {el: nums.count(el) for el in nums}

        print(cnt)

        print(max(cnt, key=cnt.get), 'lol')

        for i in range(k):
            res.append(max(cnt, key=cnt.get))
            del cnt[max(cnt, key=cnt.get)]

        print(res)
        return res

File: Data Structures & Algorithms/duplicate-integer/submission-4.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {

        return unordered_set<int>(nums.begin(), nums.end()).size() < nums.size();
        
        // std::unordered_set<int> n;

        // for (const int& num: nums) {
        //     if (n.count(num)){
        //         return true;
        //     }
        //     n.insert(num);
        // }

        // return false;
    }
};